lib/livebook/hubs/fly.ex

```diff
@@ -1,31 +1,110 @@
 defmodule Livebook.Hubs.Fly do
   @moduledoc false
-  defstruct [
-    :id,
-    :access_token,
-    :hub_name,
-    :hub_color,
-    :organization_id,
-    :organization_type,
-    :organization_name,
-    :application_id
-  ]
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Livebook.Hubs
 
   @type t :: %__MODULE__{
           id: Livebook.Utils.id(),
           access_token: String.t(),
           hub_name: String.t(),
-          hub_color: Livebook.Users.User.hex_color(),
+          hub_color: String.t(),
           organization_id: String.t(),
           organization_type: String.t(),
           organization_name: String.t(),
           application_id: String.t()
         }
 
-  def save_fly(fly, params) do
-    fly = %{fly | hub_name: params["hub_name"], hub_color: params["hub_color"]}
+  embedded_schema do
+    field(:access_token, :string)
```

Comment:
```suggestion
    field :access_token, :string
```

and we need to add ecto to `.formatter.exs`.